docs/production.md

```diff
@@ -71,7 +71,7 @@ This use case assumes that you are on a box that has SSH access to the target bo
 
 For a release version in production:
 
-    ansible-playbook -l <target-host> playbooks/katello.yml -e foreman_repositories_version=1.17 -e katello_repositories_version=3.6
+    ansible-playbook -l <target-host> playbooks/katello.yml -e foreman_repositories_version=VERSIONYOUWANT -e katello_repositories_version=VERSIONYOUWANT
```

Comment:
Perhaps it should be `WANTED_FOREMAN_VERSION` and `WANTED_KATELLO_VERSION` to indicate they're not the same value? While on the subject, perhaps it should also mention that it's up to the user to make sure the versions are compatible

Reply:
@ekohl those are both great suggestions, but I don't know how to act on them without creating another PR. Is that the way to do it?

Reply:
There is an Edit file link. For me it points to https://github.com/douglasawh/forklift/edit/patch-1/docs/production.md?pr=%2Ftheforeman%2Fforklift%2Fpull%2F1185. That's probably the easiest way.

Reply:
```suggestion
    ansible-playbook -l <target-host> playbooks/katello.yml -e foreman_repositories_version=WANTED_FOREMAN_VERSION -e katello_repositories_version=WANTED_KATELLO_VERSION
```